tests/tests.go

```diff
@@ -209,6 +210,26 @@ func initKubemacpoolParams(rangeStart, rangeEnd string) error {
 	return nil
 }
 
+func getVmFailCleanupWaitTime() (time.Duration, error) {
+	managerDeployment, err := testClient.KubeClient.AppsV1().Deployments(managerNamespace).Get(names.MANAGER_DEPLOYMENT, metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred(), "Should successfully get Deployment object")
```

Comment:
"Should successfully get manager's Deployment" better be specific to find the error at tests.